backend/testing/r_and_d/evidence_categorisation/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Outputs (all subdirectories and file types)
+outputs/
+outputs_experiments/
+!outputs/.gitkeep
+
+# Large input files (keep structure but ignore data)
+inputs/**/*.csv
+!inputs/.gitkeep
```

Comment:
I suggest placing experiment-related inputs and outputs in the experiment-specific folders. 

In your case it would be `backend/testing/r_and_d/evidence_categories/outputs` or something like that. 

That's how we've organised the other r_and_d subfolders as well.